plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -349,7 +356,7 @@ fu_plugin_thunderbolt_add (FuPlugin *plugin, GUdevDevice *device)
 	}
 	if (!is_safemode) {
 		if (fu_plugin_thunderbolt_can_update (device)) {
-			if (is_host) {
+			if (is_host && gen < 4) {
```

Comment:
so only gen <4 is allowed to be _native_, right?

Reply:
Well; it's a bit confusing.
USB4 will behave _more_ like native does but it doesn't have a differentiating element between native and legacy like the !apple implementation of Thunderbolt3 does.

Reply:
/me trusts you know what you're doing :)

Reply:
Maybe adding that comment would be good?